test/testworker.js

```diff
@@ -156,13 +156,13 @@ export default class TestWorker extends EventEmitter {
   /**
   Fetch all the common stats used by the tests.
   */
-  async fetchTaskStats(taskId, runId) {
+  async fetchTaskStats(taskId, runId, artifactPath) {
```

Comment:
Rather than passing in the artifactPath, there might be cases in the future where we need other things from the task definition.  Instead of determining the artifact path twice here, what do you think about just passing in the task definition and then we can determine the artifact path here just the one time (instead of for graphs and individual tasks)?